debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -71,9 +72,36 @@
     private static final String GET_ALL_CHANGES_FOR_TABLE = "SELECT *# FROM [#db].cdc.[fn_cdc_get_all_changes_#](?, ?, N'all update old') order by [__$start_lsn] ASC, [__$seqval] ASC, [__$operation] ASC";
     private final String get_all_changes_for_table;
     protected static final String LSN_TIMESTAMP_SELECT_STATEMENT = "TODATETIMEOFFSET([#db].sys.fn_cdc_map_lsn_to_time([__$start_lsn]), DATEPART(TZOFFSET, SYSDATETIMEOFFSET()))";
-    private static final String GET_LIST_OF_CDC_ENABLED_TABLES = "EXEC [#db].sys.sp_cdc_help_change_data_capture";
-    private static final String GET_LIST_OF_NEW_CDC_ENABLED_TABLES = "SELECT * FROM [#db].cdc.change_tables WHERE start_lsn BETWEEN ? AND ?";
-    private static final Pattern BRACKET_PATTERN = Pattern.compile("[\\[\\]]");
+
+    /**
+     * Queries the list of captured column names and their change table identifiers in the given database.
+     */
+    private static final String GET_CAPTURED_COLUMNS = "SELECT object_id, column_name" +
+            " FROM [#db].cdc.captured_columns" +
+            " ORDER BY object_id, column_id";
+
+    /**
+     * Queries the list of capture instances in the given database.
+     *
+     * If two or more capture instances with the same start LSN are available for a given source table,
```

Comment:
Out of curiosity: can there be more than two capture instances for the same source table actually? I thought I had tried that once, and SQL Server wouldn't let you create a third one.

Reply:
No, but the query doesn't rely on that (hence, two or more).

Reply:
Ok, gotcha.